src/app/components/address-input/address-input.component.spec.ts

```diff
@@ -85,12 +88,12 @@ describe('AddressInputComponent', () => {
     }
 
     function input(val: string) {
-        const instance = getAddressInputComponentInstance();
-        const inputElement = mockInput(fixture.debugElement, 'input', val);
-        instance.inputFieldFc.setValue(val);
-        instance.inputFieldFc.markAsDirty();
-        instance.inputFieldFc.markAsTouched();
-        return inputElement;
+        mockFormInput(
```

Comment:
Could you also inline the `input` function? (use `mockFormInput` directly and remove `input`) since this is what is done on the following tests, just for consistency.

Otherwise, it looks good;

Reply:
Sure. Thank you!